docs/about.md

```diff
@@ -57,24 +47,26 @@ Here's what happens when you run `ark backup create test-backup`:
 
 1. The `BackupController` makes a call to the object storage service -- for example, AWS S3 -- to upload the backup file.
 
-By default `ark backup create` makes disk snapshots of any persistent volumes. You can adjust the snapshots by specifying additional flags. See [the CLI help][30] for more information. Snapshots can be disabled with the option `--snapshot-volumes=false`.
+By default, `ark backup create` makes disk snapshots of any persistent volumes. You can adjust the snapshots by specifying additional flags. See [the CLI help][30] for more information. Snapshots can be disabled with the option `--snapshot-volumes=false`.
 
 ![19]
```

Comment:
What's this for?

Reply:
:-) Backup flow diagram
You can see it live here: https://bradamant3.github.io/ark/master/index.html

Reply:
aha!